### DISCUSSION TITLE
Any way to go out of the brackets using tab

### DISCUSSION THREAD

**Discussion by GrigoriiPrudnikov** [zed-industries/zed#13736] [Feature Requests]
I want to use tab to go out of the brackets using `tab`, instead of `arrows` like [Tab out VSCode extension](https://marketplace.visualstudio.com/items?itemName=albert.TabOut)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
